ontology/yaml/resources/HVAC/entity_types/DOAS.yaml

```diff
@@ -42,3 +42,29 @@ DOAS_EFSS_EFVSC_HTWHLSTC_SFSS_SFVSC_STC:
   - SFSS
   - SFVSC
   - STC
+
+DOAS_CHWSC_EFVSC_FDPM3X_HTWHLSTC_HWSC_RCO2M_RSPC_SFM_SFVSC_SSPC:
+  description: "Complex multi zone DOAS with heat recovery wheel."
+  is_canonical: true
+  implements:
+  - DOAS
+  - CHWSC
+  - EFVSC
+  - FDPM3X
+  - HTWHLSTC
+  - HWSC
+  - RCO2M
+  - RSPC
+  - SFM
+  - SFVSC
+  - SSPC
+
+###################################
+### Existing Non-standard Types ###
+###################################
+
+DOAS_FR_PAR_50A_1:
```

Comment:
Instead of doing this, can we instead create the canonical type and create an incomplete translation? The idea of an incomplete type is deprecated.

Reply:
following up on this, we can delete this type and reuse the other one with MISSING fields in the translation.